Assignment/Program.cs

```diff
@@ -1,7 +1,12 @@
-﻿namespace Assignment {
-   internal class Program {
-      static void Main (string[] args) {
-         Console.WriteLine ("Hello world!");
+﻿for (; ; ) {
+   Console.Write ("Enter input number :");
+   if (ulong.TryParse (Console.ReadLine (), out ulong n)) {

```

Comment:
Why ulong?

Reply:
To know the fibonacci sequence above maximum integer limit